doc/api/packages.md

```diff
@@ -924,6 +944,29 @@ require('./path/to/directory'); // This resolves to ./path/to/directory/main.js.
 When a package has an [`"exports"`][] field, this will take precedence over the
 `"main"` field when importing the package by name.
 
+### `"packageManager"`
+<!-- YAML
+added:
+  - REPLACEME
+-->
+
+* Type: {string}
+
+```json
+{
+  "packageManager": "<package manager name>@<version>"
+}
+```
+
+The `"packageManager"` field defines which package manager is expected to be
+used when working on the current project. The npm client doesn't use this field
+at the moment, but defining it to any of `yarn` or `pnpm` will let your teams
+use the exact same package manager versions without installing anything else
+than Node.js.
```

Comment:
I think we should avoid documenting which package managers are supported and which are not here, this information should be kept in `corepack.md`.
```suggestion
used when working on the current project.
```

Reply:
This hasn't been addressed. Is this an overlook or do you disagree with the suggestion?

Reply:
Overlooked! Will fix that

Reply:
Done 🚀